### PR TITLE
Staging version v4.0.1.1

### PR DESCRIPTION
**Pull request by tomasbrod**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.0.2] Unreleased
```
Comment (denravonska) [inline CHANGELOG.md]: You can leave the changelog. I have a local updated version which I will include before merging to master.

Comment (tomasbrod) [reply CHANGELOG.md]: I wont.

Comment (tomasbrod) [reply CHANGELOG.md]: But you can. I think it is appropriate to include changelog.

Comment (denravonska) [reply CHANGELOG.md]: I will overwrite it. It duplicates previous additions and it is not necessary to keep the changelog fine grained. We have a commit history for those who want to delve deeper.

Comment (tomasbrod) [reply CHANGELOG.md]: It's ok if you overwrite it. I think I removed overly detailed stuff from it, but whatever.

**Comment by tomasbrod** [comment]
Yes, @denravonska , to staging for testent. Considering the recent problems and #1366 it would be better to hold on and solve that first.

**Pull request closed without merging**